arch/src/x86_64/mod.rs

```diff
@@ -670,6 +671,13 @@ pub fn generate_common_cpuid(
                     }
                 }
             }
+            // Set Invariant TSC
+            0x8000_0007 => {
+                if !invtsc {
+                    entry.edx &= !(1 << INVARIANT_TSC_EDX_BIT);
+                }
+            }
+
```

Comment:
I believe you are "unsetting" Invariant TSC here which is also the default behavior (as `invtsc` is defaulted to `false`), right?